src/pages/404.js

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link';
+import Layout from '@/components/Layout';
+import styles from '@/components/Layout/Layout.module.scss';
+
+export default function FourOhFour() {
```

Comment:
NIT: Can we please try renaming this to something like `PageNotFound`, this will sound better